src/aks-preview/azcli_aks_live_test/ext_matrix_default.json

```diff
@@ -21,11 +21,13 @@
             "test_aks_update_azurekeyvaultsecretsprovider_with_secret_rotation",
             "test_aks_enable_addon_with_azurekeyvaultsecretsprovider",
             "test_aks_enable_addon_with_gitops",
-            "test_aks_create_with_fips"
+            "test_aks_create_with_fips",
+            "test_aks_disable_local_accounts"
         ],
         "unknown": [
             "test_aks_create_and_update_with_managed_aad_enable_azure_rbac",
-            "test_aks_create_with_virtual_node_addon"
+            "test_aks_create_with_virtual_node_addon",
```

Comment:
why exclude this?

Reply:
tracked in this [wiki]( https://dev.azure.com/msazure/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/157433/Live-Test-Failures-in-aks-preview-(with-bare-sub)) and this [work item](https://dev.azure.com/msazure/CloudNativeCompute/_workitems/edit/9997354/)